atomics/T1110.001/T1110.001.yaml

```diff
@@ -263,4 +263,39 @@ atomic_tests:
       curl -s #{remote_url} |bash
     cleanup_command: |
       rmuser -y art
-
+- name: ESXi - Brute Force Until Account Lockout
+  auto_generated_guid: f0b443ae-9565-11ee-b9d1-0242ac120002
+  description: |
+    An adversary may attempt to brute force the password of privilleged account for privilege escalation.
+    In the process, the TA may lock the account, which can be used for detection. [Reference](https://news.sophos.com/en-us/2022/07/14/blackcat-ransomware-attacks-not-merely-a-byproduct-of-bad-luck/#:~:text=A%20ransomware%20group%20attacking%20large,internal%20systems%20after%20establishing%20a)
+  supported_platforms:
+  - windows
+  input_arguments:
+    vm_host:
+      description: Specify the host name of the ESXi Server
+      type: string
+      default: atomic.local
+    plink_file:
+      description: Path to Putty
+      type: path
+      default: 'PathToAtomicsFolder\..\atomics\T1110.001\bin\plink.exe'
```

Comment:
Instead of hosting plink inside the Atomic Red Team repo, could you have the prereq download it to the ExternalPayloads directory (users will need to run the get_prereq_command one time initially to get setup to run this test). This avoids any licensing issues or other issues related to redistributing)

Reply:
It also makes the same executable available to multiple Techniques via the shared ExternalPayloads folder.